Dockerfile

```diff
@@ -34,7 +33,7 @@ LABEL org.label-schema.vcs-url=$VCS_URL
 LABEL org.label-schema.vcs-ref=$VCS_REF
 LABEL org.label-schema.version=$VERSION
 
-COPY tsconfig.json tsconfig-paths.js package.json package-lock.json* /opt/app/
+COPY tsconfig.json tsconfig-paths.js package*.json /opt/app/
```

Comment:
Most Docker files/node dependencies/ci pipelines,etc are update in our NodeJS versions upgrade Epic. Wonder why MCM wasn't added there. May need to check.

Reply:
Good point!
to be honest, IDK )
but i'll check it with Paul, and we could do it in a separate task

Reply:
MCM is not part of mojaloop at the moment.